### ISSUE TITLE
Missing dependency for kartik\yii2-helpers

### ISSUE THREAD

**Issue by mklemarczyk** [kartik-v/yii2-field-range#20]
Code below stop working after update to: v1.3.1.

``` php
<?php
use yii\helpers\Html;
use yii\widgets\ActiveForm;

use kartik\field\FieldRange;
use kartik\widgets\DatePicker;
use kartik\widgets\TimePicker;
use kartik\widgets\TouchSpin;
?>
[...]
    <?= FieldRange::widget([
        'form' => $form,
        'model' => $model,
        'label' => Yii::t('app', 'Time'),
        'attribute1' => 'time_from',
        'attribute2' => 'time_to',
        'type' => FieldRange::INPUT_WIDGET,
        'widgetClass' => TimePicker::className(),
        'useAddons' => false,
        'options1' => [
            'class' => 'form-control',
            'placeholder' => Yii::t('app', 'Start time')
        ],
        'options2' => [
            'class' => 'form-control',
            'placeholder' => Yii::t('app', 'End time')
        ],
        'widgetOptions1' => [
            'pluginOptions' => [
                'defaultTime' => '10:00',
                'minuteStep' => 5,
                'showInputs' => false,
                'showMeridian' => false,
                'template' => 'dropdown',
            ]
        ],
        'widgetOptions2' => [
            'pluginOptions' => [
                'defaultTime' => '16:00',
                'minuteStep' => 5,
                'showInputs' => false,
                'showMeridian' => false,
                'template' => 'dropdown',
            ]
        ]
    ]); ?>
[...]
```

Error is: "Class 'kartik\helpers\Html' not found"
Stack: "1. in E:\Web\root-users\meeting\vendor\kartik-v\yii2-field-range\FieldRange.php"

``` php
474 Html::addCssClass($this->labelOptions, 'control-label');
```

**Comment by mklemarczyk** [comment]
By the way, this package is also dependent on "kartik-v\yii2-widget-activeform".

**Comment by Renkas** [comment]
And the problem still persists

**Comment by mklemarczyk** [comment]
There are much more missing dependences:

```php
    /**
     * Krajee Dependent Dropdown widget [[\kartik\depdrop\DepDrop]]
     */
    const INPUT_DEPDROP = '\kartik\depdrop\DepDrop';
    /**
     * Krajee Checkbox Extended widget [[\kartik\checkbox\CheckboxX]]
     */
    const INPUT_CHECKBOXX = '\kartik\checkbox\CheckboxX';
    /**
     * Krajee Select2 widget [[\kartik\select2\Select2]]
     */
    const INPUT_SELECT2 = '\kartik\select2\Select2';
    /**
     * Krajee Typeahead widget [[\kartik\typeahead\Typeahead]]
     */
    const INPUT_TYPEAHEAD = '\kartik\typeahead\Typeahead';
    /**
     * Krajee SwitchInput widget [[\kartik\switchinput\SwitchInput]]
     */
    const INPUT_SWITCH = '\kartik\switchinput\SwitchInput';
    /**
     * Krajee TouchSpin widget [[\kartik\touchspin\TouchSpin]]
     */
    const INPUT_SPIN = '\kartik\touchspin\TouchSpin';
    /**
     * Krajee StarRating widget [[\kartik\rating\StarRating]]
     */
    const INPUT_RATING = '\kartik\rating\StarRating';
    /**
     * Krajee DatePicker widget [[\kartik\date\DatePicker]]
     */
    const INPUT_DATE = '\kartik\date\DatePicker';
    /**
     * Krajee TimePicker widget [[\kartik\time\TimePicker]]
     */
    const INPUT_TIME = '\kartik\time\TimePicker';
    /**
     * Krajee DateTimePicker widget [[\kartik\datetime\DateTimePicker]]
     */
    const INPUT_DATETIME = '\kartik\datetime\DateTimePicker';
    /**
     * Krajee Mask Money widget [[\kartik\money\MaskMoney]]
     */
    const INPUT_MONEY = '\kartik\money\MaskMoney';
    /**
     * Krajee Sortable Input widget [[\kartik\sortinput\SortableInput]]
     */
    const INPUT_SORTABLE = '\kartik\sortinput\SortableInput';
    /**
     * Krajee Treeview Input widget [[\kartik\tree\TreeViewInput]]
     */
    const INPUT_TREEVIEW = '\kartik\tree\TreeViewInput';
    /**
     * Krajee RangeInput widget [[\kartik\range\RangeInput]]
     */
    const INPUT_RANGE = '\kartik\range\RangeInput';
    /**
     * Krajee ColorInput widget [[\kartik\color\ColorInput]]
     */
    const INPUT_COLOR = '\kartik\color\ColorInput';
    /**
     * Krajee FileInput widget [[\kartik\file\FileInput]]
     */
    const INPUT_FILE_WIDGET = '\kartik\file\FileInput';
```

**Comment by Dilden** [comment]
Any chance of this being fixed by merging pull request #21?

**Comment by mklemarczyk** [comment]
@kartik-v Are you alive?

**Comment by kartik-v** [comment]
Will look into this when I get some time.

**Comment by kartik-v** [comment]
Have merged #21... as to other dependencies to all other kartik extensions and widgets... have purposely not included that dependency -- to allow folks to use this extension without needing to install the other extension. So if someone needs to use `FieldRange::INPUT_DATETIME` - he/she needs to install the `\kartik\datetime\DateTimePicker` extension.
